server/py/services/alerts/main.py

```diff
@@ -117,6 +118,10 @@ async def get_alert(
     ) -> mlrun.common.schemas.AlertConfig:
         # TODO: When alerts is a different service and not in Hydra mode, we need to send the request to the API and
         #  not access it directly (ML-8565)
+        exclude_updated = self._should_exclude_updated(
+            request.headers.get("x-mlrun-client-version")
+        )
+
```

Comment:
The comment above it is related to `ensure_project`, not this new function.
Move the function above the call to the CRUD.